### PR TITLE
Mini-challenges complete

### PR DESCRIPTION
**Pull request by Ebenezer-ebu**

### CODE REVIEW

File: src/roman-numerals/index.js [hunk 1]
```diff
@@ -4,6 +4,39 @@
...
+    let char2 = split[i+1];
+      if(val[char] < val[char2]) {
+      sum -= val[char];
+      } else {
+        sum += val[char];
+      }  
+  }
+  return sum;
+}
+console.log(romanToDecimal("I"));
```
Comment (bondz) [inline src/roman-numerals/index.js:37]: Remove these logs when submitting your PR

File: src/transpose/index.js [hunk 2]
```diff
@@ -4,6 +4,25 @@
...
  */
-function transpose(array) {}
-
+function transpose(array) {
+     if(array.length < 1 || array.length > 10) {
+      return;
+    } 
+    if(array[0].length < 1 || array[0].length > 10) {
+      return;
+    }
```
Comment (bondz) [inline src/transpose/index.js:13]: Why limit the array's length to less than 10?

That's a hardcode that you have to remove. We want our arrays to have a dynamic length.

File: src/transpose/index.js [hunk 3]
```diff
@@ -4,6 +4,25 @@
...
+          transposed[i][m] = array[m][i];
+        }   
+     } 
+    return transposed;
+}
+console.log(transpose([
+  [5, 12, 17, 9, 3],
+  [13, 4, 8, 14, 1],
+  [9, 6, 3, 7, 21],
+]));
```
Comment (bondz) [inline src/transpose/index.js:27]: Remove these logs when submitting your PR.

**Review by bondz — CHANGES_REQUESTED** [review]
Remove the hardcoded array length in transpose.

Also, try to give your variables better names and remove the log statements.

**Comment by Ebenezer-ebu** [comment]
For the transpose i thought we have some condition in the readme
Constraints
1 <= m <= 10.
1 <= n <= 10.

**Comment by Ebenezer-ebu** [comment]
Noted

**Comment by bondz** [comment]
This PR targets the wrong branch. Rather than open another PR, you can edit your PR to change the target branch. Closing.

**Pull request closed without merging**